packages/client-search/src/__tests__/integration/get-logs.test.ts

```diff
@@ -15,5 +17,18 @@ test(testSuite.testName, async () => {
     type: 'all',
   });
 
-  await expect(getLogsResponse.logs).toHaveLength(2);
+  expect(getLogsResponse.logs).toHaveLength(2);
+  const keys: Array<RequiredKeys<Log>> = [
```

Comment:
there's a TS type `Required` that does this I think:

```suggestion
  const keys: Array<Required<Log>> = [
```

Reply:
ah wait I might have misunderstood what the type is supposed to do, you want only the required keys?

Reply:
Yes, the array can only contain required keys.